### ISSUE TITLE
Fix Queries for the NavigationView.php

### ISSUE THREAD

**Issue by prestoncarman** [wwu-cs/Submitty#1]
**Describe the bug**
Should not execute queries in the view.

```
app/views/NavigationView.php:  $gradeable_ids_and_titles = $this->core->getQueries()->getAllGradeablesIdsAndTitles();
app/views/NavigationView.php:  $teams = $this->core->getQueries()->getTeamsByGradeableId($gradeable->getId());
app/views/NavigationView.php:  $this->core->getQueries()->getTeamViewedTime($graded_gradeable->getSubmitter()->getId(),$this->core->getUser()->getId()) === null) {
```

**Expected behavior**
The expected behavior is to execute the queries in the controller and pass the result to the view.

**Comment by stephenermshar** [comment]
Moved back to `in progress` since this is still broken

**Comment by stephenermshar** [comment]
I opened a duplicate pull request (#49) since the last one for this change was merged and closed. The new PR has the change split into multiple commits for each query moved into the controller. 

The breaking change is in 0273131. The error in the browser is `Notice: Undefined offset: 0 in /usr/local/submitty/site/app/libraries/database/DatabaseQueries.php on line 1804`. This error refers to the `getTeamViewedTime()` query.
